pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -277,6 +277,19 @@ func (ds *dockerService) StopPodSandbox(ctx context.Context, r *runtimeapi.StopP
 	}
 
 	// TODO: Stop all running containers in the sandbox.
+	go func() {
+		containers, err := ds.client.ListContainers()
+		if err != nil {
+			klog.Errorf("Failed to list all container: %v", err)
+			return
+		}
+		for _, v := range containers {
+			err := ds.client.StartContainer(v.ID, defaultSandboxGracePeriod)
```

Comment:
I am confused, why not `StopContainer` but `StartContainer` ?
```suggestion
			err := ds.client.StopContainer(v.ID, defaultSandboxGracePeriod)
```

Reply:
/cc @SataQiu

Reply:
Thanks for reminding.
Updated.